manifests/init.pp

```diff
@@ -198,13 +340,45 @@
   Optional[String[1]] $extract_command,
   Boolean $manage_config,
   Stdlib::Absolutepath $usershell,
-  Hash $extra_alerts    = {},
-  Hash $config_hash     = {},
-  Hash $config_defaults = {},
-  String $os            = downcase($facts['kernel']),
+  Optional[String] $web_listen_address                                          = undef,
```

Comment:
for those datatypes, please ensure empty strings are prohibited:
```suggestion
  Optional[String[1]] $web_listen_address                                       = undef,
```

Reply:
sounds good!